### PR TITLE
feat: add dashboard category accordion

### PR DESCRIPTION
**Pull request by kertuilves**
https://app.clickup.com/t/861n0qkvd

Dependencies:

- #289

### CODE REVIEW

**Comment by heshfekry** [comment]
Hey @kertuilves 

The progress bar is missing from the sub section.

image

Except for that great job on the component both on desktop and mobile i didn't see anything immediately in UX.

File: packages/cxl-lumo-styles/scss/themes/vaadin-accordion-panel.scss [hunk 1]
```diff
@@ -149,3 +149,42 @@
...
+
+:host([theme~="cxl-dashboard-category"][opened]) {
+  [part="summary"] {
+    padding-bottom: 0;
+  }
+}
+
+@media (hover: hover) {
+  :host([theme~="cxl-dashboard-category"]:not([opened])) {
+    [part="summary"]:hover {
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/scss/themes/vaadin-accordion-panel.scss:181]: I see `@media (hover: hover) {}` media query for the first time. It looks like an alternative for `:hover` but I'm confused why `:hover` needs to be repeated inside the MQ?

File: packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss [hunk 2]
```diff
@@ -265,4 +265,73 @@ cxl-vaadin-accordion {
...
+      @media #{mq.$small} {
+        padding: var(--lumo-space-l);
+      }
+
+      .entry-title {
+        margin: 0;
+      }
+
+      > span {
+        margin: 0 0 2px var(--lumo-space-m);
```
Comment (pawelkmpt) [inline packages/cxl-ui/scss/global/cxl-vaadin-accordion.scss:287]: How big impact `2px` margin has here? I tested in Chrome with `margin-bottom: 0px;` and I don't see any difference.

**Review by pawelkmpt — APPROVED** [review]
LGTM. I just left some questions

**Comment by heshfekry** [comment]
image

A little bit of padding on the horizontal scroll indicator perhaps.

**Comment by pawelkmpt** [comment]
> A little bit of padding on the horizontal scroll indicator perhaps.

Design does not have scrollbar. Maybe we'd like to hide it @heshfekry ?

**Comment by heshfekry** [comment]
@pawelkmpt hide works too. Netflix design also does not have it which is what its based of off.

File: packages/cxl-lumo-styles/scss/themes/vaadin-accordion-panel.scss (hunk 1 above)
Comment (kertuilves) [reply packages/cxl-lumo-styles/scss/themes/vaadin-accordion-panel.scss:181]: ` @media (hover: hover) {}` checks if the device supports hover and then the hover will be added if it's supported. So for example without it in mobile when someone clicks on the accordion there would be a quick background-color change which I didn't want to happen.

**Comment by pawelkmpt** [comment]
Is #289 dependency for this PR? Should we commit lightweight card so you can use here?

**Review by anoblet — APPROVED** [review]
Similar to what @pawelkmpt said, I would add https://github.com/conversionxl/aybolit/pull/289 as a dependency in the PR description:

Clickup: https://app.clickup.com/t/861n0qkvd
Dependencies:
- https://github.com/conversionxl/aybolit/pull/289

**Comment by kertuilves** [comment]
> Is #289 dependency for this PR? Should we commit lightweight card so you can use here?

I didn't see it would be necessary because it would just change the card content and it doesn't affect the layout. But if it's something that should be done I'm up for it and will do the changes.

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
Please rebase on top of `feat/dashboard` and solve conflicts.

Besides that it looks good but I'd wait for #289 and use it here in storybook instead of current `cxl-card`

**Review by pawelkmpt — APPROVED** [review]

**Pull request merged**